appdev/guides/dependencies.rst

```diff
@@ -92,45 +88,53 @@ If the library is not in the Ubuntu Repositories, but in a PPA, you can add the
 
 .. code-block:: CMake
 
-    execute_process(
-        COMMAND dpkg-architecture -qDEB_HOST_MULTIARCH
-        OUTPUT_VARIABLE ARCH_TRIPLET
-        OUTPUT_STRIP_TRAILING_WHITESPACE
-    )
-    set(SOMELIBRARY_DIR "${CMAKE_SOURCE_DIR}/build/somelib/${ARCH_TRIPLET}")
     find_package(SOMELIBRARY REQUIRED)
+
     include_directories(${SOMELIBRARY_INCLUDE_DIRS})
     target_link_libraries(mytarget ${SOMELIBRARY_LIBS})
 
-The command ``dpkg-architecture -qDEB_HOST_MULTIARCH`` is used to query the target architecture, which is part of the library build directory path, if you compiled the library with Clickable.
-
-We define the variable ``SOMELIBRARY_DIR`` with the path to the libraries build directory, to help CMake find the configuration of the library named ``SOMELIBRARY``. You may not need to do this, if you installed the library from the Ubuntu Repositories.
+The ``find_package`` command usually defines the path to the include directory as ``SOMELIBRARY_INCLUDE_DIRS`` and the library's binaries as ``SOMELIBRARY_LIBS`` (check on the library's documentation on what their CMake configuration provides exactly). Use them with the ``include_directories`` and ``target_link_libraries`` commands. See the `Camera Scanner ImageProcessing CMakeLists.txt <https://github.com/jonnius/camera-scanner/blob/master/plugins/ImageProcessing/CMakeLists.txt#L23>`_ for a real world example.
 
-The ``find_package`` command defines the path to the include directory as ``SOMELIBRARY_INCLUDE_DIRS`` and the library's binaries as ``SOMELIBRARY_LIBS``. We use those with the ``include_directories`` and ``target_link_libraries`` commands. See the `Camera Scanner ImageProcessing CMakeLists.txt <https://github.com/jonnius/camera-scanner/blob/master/plugins/ImageProcessing/CMakeLists.txt#L23>`_ for a real world example.
+At this point you should already be able to test with Clickable's desktop mode running ``clickable desktop``.
 
 Deployment
 ----------
-If you link a library statically with your app, you do not need to ship the library explicitly, as it is already inside your app binary. To do so, you usually need to compile the library yourself. Otherwise, continue with this section.
+If you linked the library statically, you can skip this step, as the library's binary is already inside your app's binary.
 
-Find out which components you need to ship. Usually this is one or more ``*.so`` (shared objects) files, linked dynamically. To get the files into the click package, you need to add an ``install`` command to your build configuration. Add the following lines to your CMakeLists.txt:
+Find out which shared object files (``*.so``) you need to ship. You can do so by simply starting the app on your device via ``clickable && clickable logs``. You should see an error message telling you which shared object file was missing.
```

Comment:
Please remove "simply" from this sentence. It trivializes steps which people are coming to the documentation to figure out.

Reply:
fair